source/portable/NetworkInterface/xilinx_ultrascale/x_emacpsif_dma.c

```diff
@@ -678,7 +684,10 @@ XStatus init_dma( xemacpsif_s * xemacpsif )
         #if ( ipconfigDRIVER_INCLUDED_RX_IP_CHECKSUM != 0 )
             value |= XEMACPS_NWCFG_RXCHKSUMEN_MASK;
         #else
-        #warning Are you sure the EMAC should not calculate incoming checksums?
+            #if ( ipconfigPORT_SUPPRESS_WARNING != 0 )
+            #warning Are you sure the EMAC should not calculate incoming checksums?
```

Comment:
Indentation

Reply:
Updated. It's modified by uncrustify. I added `{}` to make indentation right.